outputs.tf

```
@@ -7,12 +7,12 @@ output "user_pool_id" {
 }
 
 output "client_id" {
-  value     = jsondecode(data.aws_secretsmanager_secret_version.microservice_client_credentials.secret_string)["client_id"]
+  value     = jsondecode(one(data.aws_secretsmanager_secret_version.microservice_client_credentials[*].secret_string))["client_id"]
   sensitive = true
 }
 
 output "client_secret" {
-  value     = jsondecode(data.aws_secretsmanager_secret_version.microservice_client_credentials.secret_string)["client_secret"]
+  value     = jsondecode(one(data.aws_secretsmanager_secret_version.microservice_client_credentials[*].secret_string))["client_secret"]
```

Denne er jeg faktisk litt usikker på om kommer til å tryne eller ikke 🤔

Tror kanskje det er safere, i dette tilfellet (og det over) å gjøre sånn:
```suggestion
  value     = var.generate_secret ? jsondecode(data.aws_secretsmanager_secret_version.microservice_client_credentials[0].secret_string)["client_secret"] : null
```